src/AddImageChecks.cpp

```diff
@@ -93,6 +94,51 @@ class FindBuffers : public IRGraphVisitor {
     }
 };
 
+class TrimStmtToPartsThatAccessBuffers : public IRMutator {
+    bool touches_buffer = false;
+    const map<string, FindBuffers::Result> &buffers;
+
+    using IRMutator::visit;
+
+    Expr visit(const Call *op) override {
+        touches_buffer |= (buffers.find(op->name) != buffers.end());
```

Comment:
nit: would this be easier to read as `buffers.count(op->name) > 0` ?

Reply:
Interesting point. Currently in the compiler we have 77 instances of foo.find(bar) != foo.end() and a similar total number of instances of foo.count(bar) > 0, foo.count(bar) == 0, if (foo.count(bar)). Should we be changing them all? I guess it types out "foo" one fewer time.